ironfish/src/fileStores/keyStore.ts

```diff
@@ -48,10 +48,15 @@ export class KeyStore<TSchema extends Record<string, unknown>> {
 
     // Validate file store if we have a schema
     if (this.schema) {
-      const { error } = await YupUtils.tryValidate(this.schema, data)
+      const { error, result } = await YupUtils.tryValidate(this.schema, data)
+
       if (error) {
         throw new Error(error.message)
       }
+
+      if (result) {
```

Comment:
`result` and `error` are mutually exclusive on this return type right? Do we need to do this check here or can we just assign?

Reply:
Good find, I didn't see that it was typed like that.